src/components/LineChart/index.js

```diff
@@ -39,6 +39,7 @@ const propTypes = {
   onClickAnnotation: PropTypes.func,
   // event => void
   onDoubleClick: PropTypes.func,
+  onZoomXAxis: PropTypes.func,
```

Comment:
Please document what arguments this can expect to have. It looks like it doesn't get any, but that's pretty surprising to me. Could we pass back the new subDomain (x-axis) or the transformation object (or both?) or something like that?